platform/rct/Project/FreeRTOSConfig.h

```diff
@@ -159,5 +159,5 @@ standard names. */
 #define vPortSVCHandler SVC_Handler
 #define xPortPendSVHandler PendSV_Handler
 #define xPortSysTickHandler SysTick_Handler
-
+void __attribute__((used)) vTaskSwitchContext(void);
```

Comment:
Oops, snuck through.  I'm going to delete that line for now. It'll be required once we enable link time optimisation.